packages/e2e-test-utils/src/plugins.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Internal dependencies
+ */
+import { rest } from './rest-api';
+
+/**
+ * Sanitize a string to a slug.
+ * A shimmed version of WordPress's `sanitize_title()` in JavaScript.
+ *
+ * @param {string} str The input string,
+ * @return {string} The slug.
+ */
+function slugify( str ) {
```

Comment:
Hmm, is there no other util to do this? Does `kebabCase` from lodash do the job?

Reply:
It's not the same but I think should work in most cases so I changed to use `kebabCase` 👍